grails-app/services/com/jcatalog/grailsflow/ProcessManagerService.groovy

```diff
@@ -244,7 +244,7 @@ class ProcessManagerService implements InitializingBean {
      *
      * @return process ID value if process was started successfully
      */
-    public synchronized def startProcess(def processTypeID, def user, def variables) {
+    public def startProcess(def processTypeID, def user, def variables) {
```

Comment:
Looks dangerous. Should be tested heavily.

Reply:
Locally works normally: 10 start threads. 
Start process is independent part and no need to synchronize it.